apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/EarlySecurityChecks/DeviceNotGenuineDrawer.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Button, Flex, IconsLegacy, Link } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+import DrawerFooter from "./DrawerFooter";
+import { withV3StyleProvider } from "~/renderer/styles/StyleProviderV3";
+import ErrorDisplay from "../../../ErrorDisplay";
+import { createCustomErrorClass } from "@ledgerhq/errors";
+import { useHistory } from "react-router";
+import { DeviceBlocker } from "../../../DeviceAction/DeviceBlocker";
+import { setDrawer } from "~/renderer/drawers/Provider";
+import { openURL } from "~/renderer/linking";
+import { useSelector } from "react-redux";
+import { localeSelector } from "~/renderer/reducers/settings";
+import { urls } from "~/config/urls";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import { track } from "~/renderer/analytics/segment";
+
+export type Props = {
+  productName: string;
+};
+
+const DeviceNotGenuineError = createCustomErrorClass("DeviceNotGenuineError");
```

Comment:
Indeed there is no "not genuine" error existing for this case. But do you need to use `ErrorDisplay` here ? You can just display some translated text directly. 
Otherwise let's put this `DeviceNotGenuineError` in the errors package (but as it is not thrown from anywhere, i'm less a fan of that)